arcgishub/sites.py

```diff
@@ -679,6 +679,32 @@ def get(self, site_id):
             return Site(self._gis, siteItem)
         else:
             raise TypeError("Item is not a valid site or is inaccessible.")
+
+    def get_by_domain(self, domain_url):
+        """ Returns the site object for the specified domain url.
+        =======================    =============================================================
+        **Argument**               **Description**
+        -----------------------    -------------------------------------------------------------
+        domain_url                 Required string. The site url.
+        =======================    =============================================================
+        :return:
+            The site object if the item is found, None if the item is not found.
+        .. code-block:: python
+            USAGE EXAMPLE: Fetch a site successfully
+            site1 = myHub.sites.get_by_domain('opendata.dc.gov')
+            site1.item
+        """
+        if 'https' in domain_url:
+            domain_url = domain_url[7:]
```

Comment:
What would happen here if `http://` ?
There likely is a Python method for parsing a URL and getting just the domain.

Reply:
line 704